api/README.md

```diff
@@ -39,7 +31,7 @@
 - **Example**:
   GET `/earthquakes/date?start_date=01-01-2001&end_date=02-02-2002&sort=descending`
 
-### 5. Get Earthquakes by Alert Level
+### 4. Get Earthquakes by Alert Level
 - **Endpoint**: `GET /earthquakes/alert`
 - **Description**: Get earthquakes fo a specific alert level
 - **Query Parameters**:
```

Comment:
Looks good but I think the API endpoint for this is `GET /earthquakes/alert/<alert_level_colour: str> `